examples/ping_pong/Cargo.toml

```diff
@@ -1,22 +1,22 @@
 [package]
```

Comment:
This is also not formatted. I just noticed we don't enforce taplo formatting neither here nor in the nomos repo. I will fix that.

Reply:
Yes I've tried to keep the two repos as much in sync as I could, but some stuff is still not 100% the same.

Reply:
I'll double check it :)